src/audio/google/google_rtc_audio_processing.c

```diff
@@ -404,6 +404,21 @@ static int google_rtc_audio_processing_init(struct processing_module *mod)
 
 	cd->config.reference_fmt = reference_fmt.audio_fmt;
 	cd->config.output_fmt = output_fmt.audio_fmt;
+
+	/* save the base config extension */
+	size = base_cfg->base_cfg_ext.nb_input_pins * sizeof(struct ipc4_input_pin_format) +
```

Comment:
cannot assign to variable 'size' with const-qualified type 'const size_t'

Reply:
fixed. thanks!